command/whitelist/deployment/deployment.go

```diff
@@ -0,0 +1,66 @@
+package deployment
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	deploymentCmd := &cobra.Command{
+		Use:     "deployment",
+		Short:   "Top level command for updating smart contract deployment whitelist, Only accepts subcommands",
```

Comment:
Typo here, you have a capital letter after the comma

Reply:
fixed 01adfe33049ea7ea1bf55513829a168b8b4e8bbc